src/legacy/core_plugins/kibana/public/discover/_discover.scss

```
@@ -36,8 +36,18 @@ discover-app {
 }
 
 .dscResultCount {
+  display: flex;
+  justify-content: center;
   padding-top: $euiSizeXS;
   padding-left: $euiSizeM;
+
+  .dscResultHits {
+    padding-left: $euiSizeXS; 
+  }
+
+  > button {
```

Try not to use plain element selectors
```suggestion
  > .kuiLink {
```